### ISSUE TITLE
Bookmark folder option missing when editing a bookmark

### ISSUE THREAD

**Issue by srirambv** [brave/browser-ios#793]
Description:
Bookmark folder option missing when editing a bookmark

**Did you search for similar issues before submitting this one?**
Yes

**Expected behavior:**
Should show the folder option to move the bookmark into a specific folder

- Device (iPhone5, iPhone6s plus, iPad 3, ?):
iPhone 6 (iOS 10.3)

- Brave Version:
v1.4.1 (17.04.07.17)

- Steps to reproduce:
    1. Create a bookmark folder
    2. Visit any site, add it as bookmark
    3. Edit the site, bookmark folder option is missing

** Website problems only:
- did you check with Brave Shields down?
- did you check in Puffin/Dolphin/iCab/Maxthon (UIWebView-based browsers)?

- Screenshot if needed:
![image](https://cloud.githubusercontent.com/assets/17010094/24849045/b86a84ea-1dc1-11e7-804f-f8b55e412c25.png)

- Any related issues:

**Comment by jhreis** [comment]
This requires some significant UIUX changes. On desktop they show the full folder path "one / two / three" but we do not have that kind of room on mobile devices. So we have currently removed this. Folders and bookmarks need to be added into the subfolder on creation to be placed in that location.
(1. go into folder 2. create bookmark/folder)

**Comment by jhreis** [comment]
One potential, short-term option would be to only allow users to edit a bookmark/folder up or down a single directory. Not perfect, but it would solve the above concerns in the present.

@srirambv thoughts?

**Comment by jhreis** [comment]
Also see: #770

**Comment by srirambv** [comment]
I think the first level folder should be displayed. To show the complete folder structure similar to laptop I would say showing a tree structure would be nice but not sure if that is possible to overcome the iOS design of selecting from a list

**Comment by jhreis** [comment]
Yes, I think the tree structure or some nested navigation inside that menu, would probably be a good longterm solution, just doubt we will have significant time in the short term. But removing this all together could be annoying for people too. Will try to come up with something suitable in the interim.
